--- a/.venv/lib/python3.11/site-packages/szurubooru_toolkit/scripts/import_from_url.py
+++ b/.venv/lib/python3.11/site-packages/szurubooru_toolkit/scripts/import_from_url.py
@@ -183,7 +183,11 @@
                 metadata['tags'] = []

             if site == 'twitter':
-                metadata['tags'] += extract_twitter_artist(metadata)
+                artist_info = extract_twitter_artist(metadata)
+                if not all(artist_info):
+                    metadata["tags"].append(metadata["author"]["name"])
+                else:
+                    metadata['tags'] += artist_info

             with open(file, 'rb') as file_b:
                 saucenao_limit_reached = upload_media.main(
